framework/calico_framework.py

```diff
@@ -567,46 +567,51 @@ class NotEnoughResources(Exception):
 
     scheduler = TestScheduler(implicitAcknowledgements)
 
-    # Same netgroups can ping on same host
+    test_name = "Same-Host Same-Netgroups Can Ping"
     sleep_task = SleepTask(netgroups=['netgroup_a'], slave=0)
-    ping_task = PingTask(netgroups=['netgroup_a'],
-                         can_ping_targets=[sleep_task], slave=0)
-    test = TestCase([sleep_task, ping_task],
-                    name="Same-Host Different-Netgroups Can Ping One Another")
-    scheduler.tests.append(test)
+    ping_task = PingTask(netgroups=['netgroup_a'], slave=0,
+                         can_ping_targets=[sleep_task])
+    scheduler.tests.append(TestCase([sleep_task, ping_task], name=test_name))
 
-    # Same netgroups can ping across different hosts
+    test_name = "Same-Host Different-Netgroups Can't Ping"
     sleep_task = SleepTask(netgroups=['netgroup_a'], slave=0)
-    ping_task = PingTask(netgroups=['netgroup_a'],
-                         can_ping_targets=[sleep_task], slave=1)
-    test = TestCase([sleep_task, ping_task],
-                    name="Different-Host Same-Netgroups Can Ping One Another")
-    scheduler.tests.append(test)
+    ping_task = PingTask(netgroups=['netgroup_b'], slave=0,
+                         cant_ping_targets=[sleep_task])
+    scheduler.tests.append(TestCase([sleep_task, ping_task], name=test_name))
 
-    # Different netgroups cant ping on same host
+    test_name = "Different-Host Same-Netgroups Can Ping"
     sleep_task = SleepTask(netgroups=['netgroup_a'], slave=0)
-    ping_task = PingTask(netgroups=['netgroup_b'],
-                         cant_ping_targets=[sleep_task], slave=0)
-    test = TestCase([sleep_task, ping_task],
-                    name="Same-Host Different-Netgroups Can't Ping")
-    scheduler.tests.append(test)
-
-    # Tasks without calico can still use original Mesos Networking
+    ping_task = PingTask(netgroups=['netgroup_a'], slave=1,
+                         can_ping_targets=[sleep_task])
+    scheduler.tests.append(TestCase([sleep_task, ping_task], name=test_name))
+
+    test_name = "Different-Host Same-Netgroups Can Ping (Default Executor)"
```

Comment:
I modified the text formatting of Tests in this PR, but the important thing to note is the 2 new tests with `(Default Executor)` in their title.